### PR TITLE
Door

### PR DESCRIPTION
**Pull request by nkoenig**
Adds a hinged door to the world. The door will open when the robot approaches.

To test:

1. `ign launch cloudsim_sim.ign worldName:=finals_qual_demo circuit:=finals robotName1:=X1 robotConfig1:=X1_SENSOR_CONFIG_1`
2. `ign service -s /world/finals_qual_demo/set_pose --reqtype ignition.msgs.Pose --reptype ignition.msgs.Boolean --timeout 300 --req 'name: "X1", position {x:55 y:186 z:11.8}'`

### CODE REVIEW

File: subt_ign/worlds/finals_qual_demo.sdf [hunk 1]
```diff
@@ -797,6 +797,78 @@
       <uri>https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/Mine Cart</uri>
     </include>
 
+    <include>
+      <name>tile_83</name>
```
Comment (iche033) [inline subt_ign/worlds/finals_qual_demo.sdf:801]: `tile_83` exists already. I had to change this to `tile_87`

File: subt_ign/worlds/finals_qual_demo.sdf [hunk 2]
```diff
@@ -797,6 +797,78 @@
...
+    </include>
+    <include>
+      <name>wide_hinged_door</name>
+      <pose>60 186.5 11.849 0 0 0</pose>
+      <uri>https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/Wide Hinged Door</uri>
+    </include>
+    <model name="performer_detector_door">
+      <pose>55 186.5 14.3 0 0 0</pose>
+      <static>true</static>
+      <link name="link">
```
Comment (iche033) [inline subt_ign/worlds/finals_qual_demo.sdf:819]: comment out this visualization link?

**Review by iche033 — COMMENTED** [review]
there are a couple of levels with empty `<ref></ref>` (15 and 96). I had to comment them out for the world to load.

File: subt_ign/worlds/finals_qual_demo.sdf [hunk 3]
```diff
@@ -797,6 +797,78 @@
...
+            <size>2 18 3</size>
+          </box>
+        </geometry>
+        <header_data>
+          <key>type</key>
+          <value>door</value>
+        </header_data>
+      </plugin>
+      <!-- This plugin listens for the "enter" event from the performer
+           detector and starts the fog -->
```
Comment (iche033) [inline subt_ign/worlds/finals_qual_demo.sdf:849]: `...detector and opens the door -->`

File: subt_ign/worlds/finals_qual_demo.sdf (hunk 1 above)
Comment (nkoenig) [reply subt_ign/worlds/finals_qual_demo.sdf]: Fixed in 3f4a2d5

**Review by iche033 — APPROVED** [review]

**Pull request merged**